cmd/util.go

```diff
@@ -321,3 +321,15 @@ func prettifyReason(reason string) string {
 	}
 	return reason
 }
+
+func overrideMap(original map[string]string, override map[string]string) map[string]string {
+	if original == nil {
+		original = make(map[string]string)
```

Comment:
I had this discussion with @shane-lawrence and we thought it was safer not to pass around override and use a copy instead in case other parts of the program make changes to it. If you think that is a non-issue I'm happy to change it!